Client/Frontend/Browser/Browser.swift

```diff
@@ -11,6 +11,30 @@ import XCGLogger
 
 private let log = Logger.browserLogger
 
+public extension WKWebViewConfiguration {
+    public func setPlaybackPref(property: String, value: Bool) {
```

Comment:
Nit: Since we have access to stronger custom types in Swift, instead of using a String It'd be nice to use an enum here instead so we can rely on compile time checks. Strings seem error prone.